### PR TITLE
adding JERC Uncertainties module.

### PR DESCRIPTION
**Pull request by hayfasfar**

### CODE REVIEW

File: processors/HNL.py [hunk 1]
```diff
@@ -165,12 +180,24 @@
 )
 
 
+
+analyzerChain.append(
+    JetMetUncertainties(
+            era = globalOptions["year"],
+            globalTag=jecTags[globalOptions["year"]],
```
Comment (daiktas) [inline processors/HNL.py]: different indent

File: processors/HNL.py [hunk 2]
```diff
@@ -59,6 +57,28 @@
 
 minMuonPt = {2016: 25., 2017: 28., 2018: 25.}
 
+
+if isMC :
```
Comment (daiktas) [inline processors/HNL.py]: if isMC: (remove whitespace before colon)

File: processors/HNL.py [hunk 3]
```diff
@@ -59,6 +57,28 @@
...
+
+	jerTags = {2016 : 'Summer16_25nsV1_MC',
+         	   2017 : 'Fall17_V3_MC',
+            	   2018 : 'Autumn18_V7_MC'
+        }
+if args.isData : 
+
+	jecTags = {2016 : 'Summer16_07Aug2017All_V11_DATA',
+		   2017 : 'Fall17_17Nov2017_V32_DATA',
+                   2018 : 'Autumn18_V19_DATA'
```
Comment (daiktas) [inline processors/HNL.py]: indent looks inconsistent in the web browser

File: processors/HNL.py [hunk 4]
```diff
@@ -59,6 +57,28 @@
 
 minMuonPt = {2016: 25., 2017: 28., 2018: 25.}
 
+
+if isMC : 
+
+	jecTags = {2016 : 'Summer16_07Aug2017_V11_MC',
```
Comment (daiktas) [inline processors/HNL.py]: here, and below: no whitespace before colon

File: processors/HNL.py [hunk 5]
```diff
@@ -165,12 +180,23 @@
 )
 
 
+
+analyzerChain.append(
+    JetMetUncertainties(
+            era = globalOptions["year"],
+            globalTag=jecTags[globalOptions["year"]],
+	    jerTag = jerTags[globalOptions["year"]],
```
Comment (daiktas) [inline processors/HNL.py]: Also indents look inconsistent, could it be you are mixing whitespace and tabs?

Comment (hayfasfar) [reply processors/HNL.py]: That is weird. it looks fine on my github...

**Comment by daiktas** [comment]
@pep8speaks suggest diff

**Pull request merged**